### PR TITLE
Add Project Scaffold

### PR DESCRIPTION
**Pull request by benjic**
This is an attempt to add some project structure to get things off the ground. The structure was created with [stagehand][0], a utility for creating dart projects.

[0]: https://pub.dartlang.org/packages/stagehand

### CODE REVIEW

**Comment by benjic** [comment]
Did you want to do +1's for code review on pull requests, @falkzach ?

File: pubspec.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+name: 'client'
+version: 0.0.1
+description: An absolute bare-bones web app.
+authors:
+  - Zach Faulkner
```
Comment (benjic) [inline pubspec.yaml]: I didn't add your email address to respect your privacy. Add it if you want.

File: test/client_test.dart [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+void main() {
+  group('Awesome', () {
+    test('should always be awesome', () {
+      Awesome awesome = new Awesome();
+
+      expect(awesome.isAwesome, isTrue);
+    });
+  });
+}
```
Comment (benjic) [inline test/client_test.dart]: This is an example of the test runner and tests.

File: web/main.dart [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import 'dart:html';
+
+import 'package:client/client.dart';
+
+void main() {
+  Awesome you = new Awesome();
+
+  querySelector('#output').text =
+      you.isAwesome ? 'You awesome!' : 'You could be awesome';
+}
```
Comment (benjic) [inline web/main.dart]: Boilerplate entry point.

**Comment by falkzach** [comment]
LGTM Merged

**Pull request merged**